packages/experiments/src/components/Card/CardItem/CardItem.tsx

```diff
@@ -0,0 +1,27 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { withSlots, createComponent, getSlots } from '../../../Foundation';
+import { ICardItemComponent, ICardItemProps, ICardItemSlots } from './CardItem.types';
+import { styles } from './CardItem.styles';
+
+const view: ICardItemComponent['view'] = props => {
+  const childNodes: React.ReactElement<{}>[] = React.Children.toArray(props.children) as React.ReactElement<{}>[];
+  const first = childNodes[0];
+  if (!first) {
+    return null;
+  }
```

Comment:
Hmm, this seems to be a copy of the old impl as opposed to using `React.Children.count()`

Reply:
Yeah, will change this.